sdk/python/lib/pulumi/provider/experimental/server.py

```diff
@@ -0,0 +1,705 @@
+# Copyright 2025, Pulumi Corporation.
```

Comment:
Is this a copy of `sdk/python/lib/pulumi/provider/server.py` but using `PropertyValue` ? Is the plan to entirely replace the old provider with this one?

Reply:
Yes plus filling in the remaining methods